tilecloud_chain/__init__.py

```diff
@@ -381,7 +381,7 @@ def _resolution_scale(self, resolutions):
 
     @staticmethod
     def _configure_logging(options, format_):
-        if os.environ.get('CI', 'FALSE') == 'TRUE':
+        if os.environ.get('CI', 'TRUE') == 'TRUE':
```

Comment:
Looks wrong ...

Reply:
Indeed! I've missed the fact this is production code...